server/lib/git.coffee

```diff
@@ -113,3 +115,20 @@ module.exports.update = (app, callback) ->
         env: "USER": app.user
 
     executeUntilEmpty commands, config, callback
+
+###
+    Change branch of <app>
+###
+module.exports.changeBranch = (app, newBranch, callback) ->
+
+    # Setup the git commands to be executed
+    commands = [
+        "git fetch origin #{newBranch}:#{newBranch} && " + \
```

Comment:
why do you use `&&` there. The `executeUntilEmpty` function runs all command in the array.